xbmc/video/PlayerController.cpp

```diff
@@ -224,19 +224,23 @@ bool CPlayerController::OnAction(const CAction &action)
         if (++currentAudio >= audioStreamCount)
           currentAudio = 0;
         appPlayer->SetAudioStream(currentAudio); // Set the audio stream to the one selected
-        std::string aud;
+
         std::string lan;
         AudioStreamInfo info;
         appPlayer->GetAudioStreamInfo(currentAudio, info);
         if (!g_LangCodeExpander.Lookup(info.language, lan))
           lan = g_localizeStrings.Get(13205); // Unknown
-        if (info.name.empty())
-          aud = lan;
-        else
-          aud = StringUtils::Format("{} - {}", lan, info.name);
+
+        std::string textInfo = lan;
+        if (!info.name.empty())
+          textInfo += " - " + info.name;
+        if (!info.codecDesc.empty())
+          textInfo += " (" + info.codecDesc + ")";
```

Comment:
Maybe the format of the info should be localizable instead of hard coded? In other languages the dash (-) might not be the preferred delimiter for example.

Reply:
I just adapted the existing code that was already hardcoded with dash char,
there are many parts of code, not only this one, that could be localizable, but I don't adjust them on this PR, its already big

Reply:
Not sure this is a wise discussion. After your PR got merged nobody will clean it up i fear.

Reply:
this code (and also the others "next"/"cycle" behaviours imo must be full reworked,
because now will result that stream dialogs have applied a sort order, while these do not, and so users "scroll" unordered streams

so the idea is use the new helper file added here to get an ordered list of streams (done with SubtitleStreamInfo, etc...) and so after apply the "next"/"cycle" behaviour, along with this changes can be add new localizables id's

if you really want I will do here, but it will result in very mess PR
since all these changes that are not few, are not directly related to dialogs implementations

Reply:
in my previous message i said "many parts of code",
because there are also other situations,
for example on skins xml, the text on `$INFO` is not localized
example:
https://github.com/xbmc/xbmc/blob/12ac3e2085665ae0f23fcb719e0aa67cbd433352/addons/skin.estuary/xml/DialogPlayerProcessInfo.xml#L240-L249
and same thing on the xml code that i add on this PR

unless i am not aware of it,
suggest that kodi core dont provide a kind of `$INFOLOC` "localizable" where you can inject the values to be added to the string as arguments directly from the xml,
this forces to apply localizations via source code, and does not allow provide clean values to skins, for example you have:
`Player.Process(audiochannels)` that provide the n° of channels
now to localize it, you forced to localize value as string from source code and will result in a string e.g.:
"channels: 4"
but in this way skins cant have a clean value of "4"
where they can manage as they wish